src/Components/PlaylistModal/playlist.css

```diff
@@ -0,0 +1,51 @@
+.modal-wrap {
+    background: #342e2ebf;
+    width: 100vw;
+    position: fixed;
+    height: 100vh;
+    z-index: 35;
+}
+
+.flex-center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
Maybe make a class mainly in root for:
display : flex
justify-content : center
align-items  : center 
and the use it in different places because it is gonna be used to different places. it will come in handy.